config/routes.rb

```diff
@@ -21,5 +21,8 @@
     namespace :v0 do
       resources :ping, only: [:index], constraints: { format: 'json' }
     end
```

Comment:
This `v0` is unnecessary - it was for doing a very simple API endpoint in Cooper.